app/src/main/res/drawable/splash_screen.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item>
+        <color android:color="@color/colorPrimary"/>
+    </item>
+
+    <!--<item>
```

Comment:
That's great that our splash is only a `bg` with our primary color. I wouldn't think it is good to include the bitmap... so you can remove this commented element.

Why I don't like having a splash (at least with an image)? 
A: we can't see it in Telegram, Whats App and Slack. But maybe @rafaelks can tell us if our mobile app will have a splash screen (with image and so on...).

Old but gold: 
https://cyrilmottier.com/2012/05/03/splash-screens-are-evil-dont-use-them/
https://www.bignerdranch.com/blog/splash-screens-the-right-way/

Reply:
https://material.io/guidelines/patterns/launch-screens.html

Reply:
Nice example! From the link we can see how Gmail actually does: it shows the branded launch screen at the first opening and after it, when we open the app it is not show anymore. In that way, I believe that you can remove the commented element (it isn't in use).